### PR TITLE
Update users database from CRM 

### PR DESCRIPTION
**Pull request by VadimDyachenko**
Connect #56

### CODE REVIEW

File: src/main/java/juja/microservices/users/service/UserService.java [hunk 1]
```diff
@@ -65,15 +86,63 @@ public UserService(UserRepository userRepository) {
...
-    private UserDTO convertUserToUserDto(User user) {
-        return new UserDTO(user.getUuid(), user.getSlack(), user.getSkype(), user.getFullName());
+    @Scheduled(cron = "${cron.expression}")
+    public void scheduleUpdateUsers() {
+        updateUsersFromCRM();
+    }
+
+    public List<UserDTO> updateUsersFromCRM() {
+        Long lastUpdate = repository.findMaxLastUpdate();
+        if (lastUpdate == null) lastUpdate = 0L;
```
Comment (OlgaKulikova) [inline src/main/java/juja/microservices/users/service/UserService.java]: I don't like such approach to write (in one row), and this is not on code convention.

File: src/main/java/juja/microservices/users/service/UserService.java [hunk 2]
```diff
@@ -65,15 +86,63 @@ public UserService(UserRepository userRepository) {
...
+
+        logger.info("Starting update users database. Last update was at {}",
+                LocalDateTime.ofEpochSecond(lastUpdate, 0, OffsetDateTime.now().getOffset()));
+
+        return getConvertedResult(updateUsersDatabase(getUpdatedUsersFromCRM(lastUpdate)));
```
Comment (OlgaKulikova) [inline src/main/java/juja/microservices/users/service/UserService.java]: short form of writing could be the resource of errors that is difficult to find and read code. Too long chain of invocation several methods. Please write separately.

Comment (VadimDyachenko) [reply src/main/java/juja/microservices/users/service/UserService.java]: Done! But I think my new version looks like a "Pascal style" :)

File: src/main/java/juja/microservices/users/service/UserService.java (hunk 1 above)
Comment (VadimDyachenko) [reply src/main/java/juja/microservices/users/service/UserService.java]: Done! I extracted it to the separate method

**Review by OlgaKulikova — APPROVED** [review]

**Pull request merged**